integration/configtx/configtx_test.go

```diff
@@ -140,81 +140,190 @@ var _ = Describe("ConfigTx", func() {
 		Eventually(createChannel, network.EventuallyTimeout).Should(Equal(0))
 
 		By("joining all peers to the channel")
+		testPeers := network.PeersWithChannel("testchannel")
 		network.JoinChannel("testchannel", orderer, testPeers...)
 
+		By("getting the current channel config")
+		org2peer0 := network.Peer("Org2", "peer0")
+		channelConfig := nwo.GetConfig(network, org2peer0, orderer, "testchannel")
+		c := configtx.New(channelConfig)
+
+		By("updating orderer channel configuration")
+		o := c.Orderer()
+		oConfig, err := o.Configuration()
+		Expect(err).NotTo(HaveOccurred())
+		oConfig.BatchTimeout = 2 * time.Second
+		err = o.SetConfiguration(oConfig)
+		Expect(err).NotTo(HaveOccurred())
+		host, port := ordererHostPort(network, orderer)
+		err = o.Organization(orderer.Organization).SetEndpoint(configtx.Address{Host: host, Port: port + 1})
```

Comment:
Did you really want to use `port + 1` or get another reserved port from the network?

Reply:
I believe the idea was that this port wasn't actually going to be used for anything hence no need to get another reserved port. I think this is fine as is.